new resource `azurerm_elastic_san_volume` (#24802)
hashicorp/terraform-provider-azurerm: website/docs/r/elastic_san_volume.html.markdown

  name                 = "example-disk"
  location             = azurerm_resource_group.example.location
  resource_group_name  = azurerm_resource_group.example.name
  create_option        = "Empty"
  storage_account_type = "Standard_LRS"
  disk_size_gb         = 2
}

resource "azurerm_snapshot" "example" {
  name                = "example-ss"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  create_option       = "Copy"
  source_uri          = azurerm_managed_disk.example.id
}

resource "azurerm_elastic_san_volume" "example2" {
  name            = "example-esv2"
  volume_group_id = azurerm_elastic_san_volume_group.example.id
  size_in_gib     = 2
  create_source {
    source_type = "DiskSnapshot"
    source_id   = azurerm_snapshot.example.id
  }
}
```

## Arguments Reference

The following arguments are supported:

* `name` - (Required) Specifies the name of this Elastic SAN Volume. Changing this forces a new resource to be created.

* `volume_group_id` - (Required) Specifies the Volume Group ID within which this Elastic SAN Volume should exist. Changing this forces a new resource to be created.

> * `size_in_gib` - (Required) Specifies the size of the Elastic SAN Volume in GiB. The size should be within the remaining capacity of the parent Elastic SAN. Possible value is integer between `1` and `65536` (16 TiB).

Reviewer: ```suggestion
* `size_in_gib` - (Required) Specifies the size of the Elastic SAN Volume in GiB. The size should be within the remaining capacity of the parent Elastic SAN. Possible values are between `1` and `65536` (16 TiB).
```
